recruiter/views.py

```diff
@@ -71,4 +48,5 @@ def recruiterRegister(request):
 
 
 def recruiter_created_successfully(request):
+    jobboard.views.add_navbar_links_to_context(request, )
```

Comment:
This does nothing as the function needs to get a dictionary as well.
Anyway, I already have a PR open about this so you can remove this line.

Reply:
fixed it